airflow/contrib/hooks/gcs_hook.py

```diff
@@ -193,16 +187,7 @@ def upload(self, bucket, object, filename,
         :type mime_type: str
         :param gzip: Option to compress file for upload
         :type gzip: bool
-        :param multipart: If True, the upload will be split into multiple HTTP requests. The
```

Comment:
Does this mean that multipart support is gone?

Reply:
@Fokko No, Google client library handles the multipart for your now. 

https://github.com/googleapis/google-cloud-python/blob/11c543ce7dd1d804688163bc7895cf592feb445f/storage/google/cloud/storage/blob.py#L989-L997

Reply:
I can add that as well in `Updating.md` if you think other users might think the same as well.

Reply:
It's a change in API of the operator so should go in UPDATING, yes.

Reply:
Done

Reply:
@Fokko Added a comment on `Multipart`  in Updating.md.